sources/packages/api/src-sql/Restrictions/Create-institution-restrictions.sql

```diff
@@ -0,0 +1,41 @@
+-- Create Table sims.institution_restrictions
+CREATE TABLE IF NOT EXISTS sims.institution_restrictions(
+    id SERIAL PRIMARY KEY,
+    institution_id INT NOT NULL REFERENCES sims.institutions(id),
+    restriction_id INT NOT NULL REFERENCES sims.restrictions(id),
+    restriction_note_id INT REFERENCES sims.notes(id),
```

Comment:
juts to check, is these many to many with notes?

Reply:
like student restrictions it is One to One with Note.